Account.java

```diff
@@ -0,0 +1,93 @@
+package doubleEntry;
+
+import doubleEntryComponents.Agent;
+import doubleEntryComponents.actions.Action;
+import doubleEntryComponents.contracts.Contract;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Account {
+
+    public Account(String name, AccountType accountType, Double startingBalance) {
+        this.name = name;
+        this.accountType = accountType;
+        this.total = startingBalance;
+        this.contractClass = null;
```

Comment:
Initially, I thought that each Account should include a single contract Type inside (for example, a given account can only hold Bonds, and another can only hold Repos). However, this condition is never enforced and `contractClass` is not used.